status-im/status-react
# [FIX #4362] Updates App Icons

[comment]: # (Please replace ... with your information. Remove < and >)
[comment]: # (To auto-close issue on merge, please insert the related issue number after # i.e fixes #566)

Fixes #4362 

### Summary:

[comment]: # (Summarise the problem and how the pull request solves it)
Updates App icons to those specified in #4362 

### Steps to test:
- Install Status
- Check if icons reflect updated assets



status: ready
